src/main/java/com/powsybl/openloadflow/OpenLoadFlowParameters.java

```diff
@@ -170,9 +174,13 @@ public enum SlackDistributionFailureBehavior {
 
     public static final String MAX_PLAUSIBLE_TARGET_VOLTAGE_PARAM_NAME = "maxPlausibleTargetVoltage";
 
-    public static final String MIN_REALISTIC_VOLTAGE_PARAM_NAME = "minRealisticVoltage";
+    public static final String MIN_REALISTIC_VOLTAGE_NEWTON_RAPHSON_PARAM_NAME = "minRealisticVoltageNewtonRaphson";
+
+    public static final String MAX_REALISTIC_VOLTAGE_NEWTON_RAPHSON_PARAM_NAME = "maxRealisticVoltageNewtonRaphson";
+
+    public static final String MIN_REALISTIC_VOLTAGE_KNITRO_SOLVER_PARAM_NAME = "minRealisticVoltageKnitroSolver";
 
-    public static final String MAX_REALISTIC_VOLTAGE_PARAM_NAME = "maxRealisticVoltage";
+    public static final String MAX_REALISTIC_VOLTAGE_KNITRO_SOLVER_PARAM_NAME = "maxRealisticVoltageKnitroSolver";
```

Comment:
why can't we use same parameters for both solvers ?

Reply:
We could but we wanted to set a 1.5 upper bound because the NR upper bound (2.0) was too large most of the time and also complicates Knitro's convergence

Reply:
ok understood, but there is a confusion on usage of this parameter.
`minRealisticVoltage` / `maxRealisticVoltage` are used to **check** solution after solve, are not used as variable **bounds**.

I propose that:
- you leave `minRealisticVoltage` / `maxRealisticVoltage` as they are
- you may or may not use `minRealisticVoltage` / `maxRealisticVoltage` as a solution check for Knitro solver, up to you to decide whether this is relevant or not
- you can introduce new parameters for voltage magnitude bounds, but with a different naming so that no confusion is made, e.g. `knitroLowVoltageBound` `knitroHighVoltageBound`

Reply:
Okay it makes sense , thank you!